http-server-microservice-provider/src/main/java/io/silverware/microservices/providers/http/HttpServerMicroserviceProvider.java

```diff
@@ -67,7 +67,7 @@
    public void initialize(final Context context) {
       this.context = context;
       context.getProperties().putIfAbsent(HTTP_SERVER_PORT, 8080);
-      context.getProperties().putIfAbsent(HTTP_SERVER_ADDRESS, "localhost");
+      context.getProperties().putIfAbsent(HTTP_SERVER_ADDRESS, "0.0.0.0");
```

Comment:
When you specify localhost in a URL, Camel exposes the endpoint only on the local TCP/IP network interface, so it cannot be accessed from outside the machine it operates on.
If you need to expose a Jetty endpoint on a specific network interface, the numerical IP address of this interface should be used as the host. If you need to expose a Jetty endpoint on all network interfaces, the 0.0.0.0 address should be used.
Source:
http://camel.apache.org/undertow.html